ARKitExample.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "self:/Users/ceagleston/Development/video-quickstart-swift/ARKitExample/ARKitExample.xcodeproj">
```

Comment:
Uh.. You might want to revisit this, @ceaglest

Reply:
Whoops. Let me clean this up. 👍